plutus-tx/CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Revision history for `plutus-tx`
+
+This format is based on [Keep A Changelog](https://keepachangelog.com/en/1.0.0).
+
+## 0.2.0.0 -- 2021-12-13
+
+### Added
+
+* Specialized `abs` and `negate` for new `Rational` type, using much less
+  on-chain space.
+
+### Changed
+
+* New implementation of `Rational`, which is more efficient on-chain.
+* `abs`, `divMod` and `quotRem` now in `PlutusTx.Numeric`, as they're not
+  specific to rational numbers.
+
+### Removed
+
+* `Ratio` data structure.
```

Comment:
This feels a bit needlessly breaking, although I checked and at least all uses in `plutus-apps` are `Ratio Integer`. I think I'm *probably* fine about it.

Reply:
The current design is creating on-chain space usage for no reason. Nothing _other_ than `Integer` can currently go into it.

Reply:
Other things *could* go in it. Nothing currently does. Anyway, I'm fine with removing it.